python/pyarrow/tests/test_cuda.py

```diff
@@ -792,3 +792,155 @@ def test_IPC(size):
     p.start()
     p.join()
     assert p.exitcode == 0
+
+
+def _arr_copy_to_host(carr):
+    # TODO replace below with copy to device when exposed in python
```

Comment:
Is there a GH issue for this?

Reply:
Not yet (I still need to create a whole set of issues to track the work I am planning the next months to improve the Device / MemoryManager bindings in Python)